### ISSUE TITLE
Can you share the hyper-parameter "rounds" for reproducing the resulting images?

### ISSUE THREAD

**Issue by RamonYeung** [E-Renshaw/ping-pong-gan#1]
Can you share the hyper-parameter "rounds" for reproducing the resulting images?

I found your implementation really helps me understanding the ping-pong training idea. I would like to reproduce the results. It seems like the "rounds" is a list of training epoches for birds and houses. 

It is hard to tune so can you please share some settings as a starting point? Thanks a lot in advance!

**Comment by fathersteph34** [comment]
Yes I have the same question with 'rounds', looking forward for answers. x
